lib/active_model/introspector/lookups.rb

```diff
@@ -17,7 +17,7 @@ def each
 
       private
 
-      ONLY_ALL_CAPS_CONSTS = /^[A-Z_]*$/
+      ONLY_ALL_CAPS_CONSTS = /^[A-Z0-9_]*$/.freeze
```

Comment:
Should this have a test to catch this in the future?

Reply:
No. It's going away.